shared/chat/conversation/messages/react-button/emoji-picker/index.tsx

```diff
@@ -226,7 +226,7 @@ class EmojiPicker extends React.PureComponent<Props, State> {
         key={emoji.short_name}
       >
         {emoji.source ? (
-          <Kb.CustomEmoji size="Medium" src={emoji.source} alias={emoji.short_name} />
+          <Kb.CustomEmoji size="MediumLarge" src={emoji.source} />
```

Comment:
Does mobile need to be different? `singleEmojiWidht` (used below for stock emojis) is different on mobile vs desktop.

Reply:
This PR doesn't handle mobile changes, was still running into some alignment issues on ios and android so I figured i'd just get desktop changes in for now